gemhardware/devices/src/common/amc/HwGenericAMC.cc

```diff
@@ -88,12 +88,16 @@ bool gem::hw::HwGenericAMC::isHwConnected()
     return true;
   } else if (gem::hw::GEMHwDevice::isHwConnected()) {
     CMSGEMOS_INFO("basic check: HwGenericAMC pointer valid");
-    m_maxLinks = this->getSupportedOptoHybrids();
+    //FIXME apparently this->getSupportedOptoHybrids(); doesn't work!!!
+    m_maxLinks = 12;
+    //m_maxLinks = this->getSupportedOptoHybrids();
```

Comment:
Can we just fix this now rather than committing the hardcoded magic?
Or what was the reason for the failure? (i.e., was it reported in an issue somewhere?)

Reply:
Issue opened here https://github.com/cms-gem-daq-project/cmsgemos/issues/317